adventuregame.rb

```diff
@@ -1,73 +1,109 @@
+# frozen_string_literal: true
+
+# == Adventure \Game
+#
+# This is gonna be such a great game! Here's some details...
+# - TODO: add details
+#
+# == Running the game
+#
+# To run the game, ... (TODO: moar details needed; etc.)
+
 #GAME START
+
+# Player
+#
+# TODO: describe what this class is/does/is-responsible-for
+#
+# TODO: you might add an example of using this class here, too
 class Player
-  @hit_points
-  @attack_power
-  @chance_to_hit
-  
+  attr_accessor :hit_points
+  attr_accessor :attack_power
+  attr_accessor :chance_to_hit
+
   MAX_HIT_POINTS = 100
-  
+
+  # A predicate method to check whether a Player is alive or not
+  # - returns: Boolean (true/false)
```

Comment:
_PUBLIC_ methods represent your new Class' API (Application Programming Interface). As such, they are literally the list of messages you can send to an instance of this Class. They should be documented...
- TODO: 1) Describe what it does...
- TODO: 2) it's helpful to document the expected return values...